src/Types/ContractEvent.cs

```diff
@@ -9,4 +12,31 @@ public sealed record ContractEvent(byte[] Bytes)
     /// Return hex representation.
     /// </summary>
     public string ToHexString() => Convert.ToHexString(this.Bytes).ToLowerInvariant();
+
+    /// <summary>
+    /// Deserialize event from <see cref="schema"/>.
+    /// </summary>
+    /// <param name="schema">Module schema in hexadecimal.</param>
+    /// <param name="contractName">Contract name.</param>
+    /// <returns><see cref="Bytes"/> deserialized.</returns>
+    /// <exception cref="InteropBindingException">Thrown when event wasn't able to be deserialized from schema.</exception>
+    public string? GetDeserializeEvent(
```

Comment:
**Suggestion:** The output should indicate that this is JSON instead of just `string`, maybe [`Utf8JsonReader`](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/use-utf8jsonreader) is a good match, since it won't actually deserialize the JSON string and the user can bring their own class for this.

Reply:
Sum up of discussion p2p. It would be nice to show output is json but since `Utf8JsonReader` is a `ref struct` it is limited how it can be used. We discussed `JsonNode` or `JsonDocument` but in the end it will be a overhead.

We will write it clearly in the documentation and then let it be up to the user to parse the result as efficient as they like.